mllib/src/main/scala/org/apache/spark/ml/clustering/LDA.scala

```diff
@@ -490,7 +490,7 @@ abstract class LDAModel private[ml] (
         Vectors.zeros(k)
       } else {
         val (ids: List[Int], cts: Array[Double]) = vector match {
-          case v: DenseVector => ((0 until v.size).toList, v.values)
+          case v: DenseVector => (List.range(0, v.size), v.values)
```

Comment:
Please update the PR title or remove this from this PR.

Reply:
I think this is OK, as it mirrors a similar change (optimization) we made in the previous PR. You're right it's orthogonal to the typo fix, but will be fine to add.